ts/features/itwallet/onboarding/screens/WalletCardOnboardingScreen.tsx

```diff
@@ -66,10 +75,12 @@ const activeBadge: Badge = {
 const isUpcomingCredential = (type: string): boolean =>
   upcomingCredentials.includes(type as any);
 
+const isNewCredential = (type: string): boolean =>
+  newCredentials.includes(type as any);
```

Comment:
Can we remove `any`?

Reply:
Should be fixed with https://github.com/pagopa/io-app/pull/7259/commits/cf2ad82bc96adb5806d1c716cb01c420f17c35ee, thanks!